libafl/src/bolts/llmp.rs

```diff
@@ -732,7 +730,7 @@ where
     /// Ref to the last message this sender sent on the last page.
     /// If null, a new page (just) started.
     pub last_msg_sent: *const LlmpMsg,
-    /// A vec of page wrappers, each containing an intialized AfShmem
```

Comment:
Could be ```[`ShMem`]```

Reply:
Agree. Will change it.

Reply:
Fixed in 0322de43768d68e5c1d516c849234ca7d789a49c